tests/unittests/config/test_schema.py

```diff
@@ -1806,6 +1924,6 @@ def test_handle_schema_args_annotate_deprecated_config(
         )
         handle_schema_args("unused", args)
         out, err = capsys.readouterr()
-        assert expected_output.format(cfg_file=user_data_fn) == out
+        assert expected_output.format(user_data_fn).split() == out.split()
         assert not err
         assert "deprec" not in caplog.text
```

Comment:
I know this is not part of this PR, but just for the record.

This check might not be sufficient, as cloud-init logs the following line, containing a full path to the jsonschema and potentially any valid string for a path:

```log
2023-02-15 11:54:06 DEBUG     cloudinit.util:util.py:1531 Reading from /home/aciba/canonical/worktrees/cloud-init/holmanb/deprecation-schema/cloudinit/config/schemas/schema-cloud-config-v1.json (quiet=False)\n
```